cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/BrokerCapacityStats.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.servlet.response.stats;
+
+import com.linkedin.kafka.cruisecontrol.config.KafkaCruiseControlConfig;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * Get broker level stats related to capacity in human readable format.
+ */
+public class BrokerCapacityStats extends BrokerStats {
+  private final List<SingleBrokerCapacityStats> _brokerCapacityStats;
+
+  public BrokerCapacityStats(KafkaCruiseControlConfig config) {
+    super(config);
+    _brokerCapacityStats = new ArrayList<>();
+  }
+
+  public void addSingleBrokerCapacityStats(String host, int id, boolean isEstimated, double diskCap, double cpuCap, double nwInCap,
+                                            double nwOutCap) {
+    SingleBrokerCapacityStats singleBrokerCapacityStats = 
+        new SingleBrokerCapacityStats(host, id, isEstimated, diskCap, cpuCap, nwInCap, nwOutCap);
+    _brokerCapacityStats.add(singleBrokerCapacityStats);
+    _isBrokerStatsEstimated = _isBrokerStatsEstimated || isEstimated;
+    _hostFieldLength = Math.max(_hostFieldLength, host.length());
+  }
+
+  @Override
+  protected void discardIrrelevantResponse() {
+    // Discard irrelevant response.
```

Comment:
miss `super.discardIrrelevantResponse()`?

Reply:
the parent `BrokerStats` defined method for `discardIrrelevantResponse` has a parameter for CC parameters. The regular `discardIrrelevantResponse()` is initialized as abstract so `super.discardIrrelevantResponse()` should be unnecessary.

Reply:
I realized the way how I did this was pretty bad, it had 2 methods where `discardIrrelevantResponse()` was called from another function, `discardIrrelevantAndCacheRelevant()`, and was overloaded by the children classes but it led to more confusion. I modified the function to just be defined in the children classes so the logic is more centralized.